functions/exercise20/script.js

```diff
@@ -9,4 +9,11 @@ function getStringIdOfRandomChars(specifiedLength) {
         randomCharacters += stringCharacters[randomIndex];
     }
     return randomCharacters;
+}
+
+
+function showStringIdOfRandomChars() {
```

Comment:
What is "string id" of random characters? Please rename it to something more meaningful.